CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ Main (unreleased)
 ### Bugfixes
 
 - Fixed a bug in `import.git` which caused a `"non-fast-forward update"` error message. (@ptodev)
+- Fixed a bug in `azure_event_hubs` parser for messages without `time` field. (@andriikushch)
```

Comment:
```suggestion
- Support logs that have a `timestamp` field instead of a `time` field for the `loki.source.azure_event_hubs` component. (@andriikushch)
```
I think that your changelog entry is a bit obscure for the users. I suggested an alternative, feel free to modify as you want. Also maybe I would put it in enhancement and not bugfix

Reply:
Thank you @wildum! I will merge the suggestion and move it to the enhancement section